index.html

```diff
@@ -57,5 +58,204 @@ <h3 class="contact-title">let's connect</h3>
 				</div>
 			</div>
 		</header>
+
+		<!-- Work section -->
+		<section class="work">
+			<div class="single-card">
+				<img
+					src="/images/SnapshootPortfolio-1.png"
+					alt="Portfolio 1 image"
+					class="p-img"
+				/>
```

Comment:
- [ ] This image is currently showing just the `alt` text. I can see the path to the image is correct but it's missing a `.` at the beginning. Let's add the dot and see how the images magically show up. 😉 
`./images/SnapshootPortfolio-1.png`